python/python/glide/constants.py

```diff
@@ -42,3 +42,12 @@
         Union[bytes, List[Mapping[bytes, Union[bytes, Set[bytes]]]]],
     ]
 ]
+TFunctionStatsResponse = Mapping[
```

Comment:
I think we need to export it. See `__init__.py` file

Reply:
I asked @aaron-congo about this and it seems like the other types from constants.py aren't in there, aside from `OK`. I guess we don't need it there?

Reply:
`OK`
I trust @aaron-congo

Reply:
@Yury-Fridlyand Only OK is being exported from this file, even though other types are used in the function signatures in core.py etc. I'm not sure if that is accidental or intentional, we should probably ask AWS